doc/exception_interrupts.rst

```diff
@@ -94,13 +98,14 @@ Ibex can trigger an exception due to the following exception causes:
 The illegal instruction exception, instruction access fault, LSU error exceptions and ECALL instruction exceptions cannot be disabled and are always active.
 
 
-Handling
---------
+Nested Interrupt/Exception Handling
+-----------------------------------
 
 Ibex does support nested interrupt/exception handling.
```

Comment:
Could you specify what the max number of nested interrupts would be that Ibex can support?

Reply:
Well, since the saving of these registers is has to happen in software, the number of nested interrupts supported is infinite.

Reply:
Sorry, let me clarify what I mean, I phrased that question poorly. At just the hardware level, Ibex has the standard set of CSRs to handle CSRs as well as the `mstack` CSRs to support one level of nesting. Say that the core is currently handling a nested interrupt so both of these sets of CSRs contain relevant state, and then the core receives another nested interrupt in the middle. Assuming software doesn't store this state separately, it seems like the stored state from the initial interrupt would get overridden, correct?
Would this mean that the core can support at most one nested interrupt at the hardware level before having to overwrite state (assuming software doesn't do anything to help)?
Please feel free to correct me if any of this is inaccurate, as the spec is very vague on nested interrupts and related topics.

Reply:
The `mstack` CSRs are only there for the NMI. The other CSRs are only ever restored from them when executing an MRET in NMI mode. They are not accessible by software. Thus the hardware "nesting" is only supported for one NMI. I tried to clarify this in the documentation.